sample-input/bundled-lattice/bundled-lattice.py

```diff
@@ -38,13 +35,13 @@
 
 log.py_printf('NORMAL', 'Creating surfaces...')
 
-circles = []
-planes = []
+circles = list()
+planes = list()
```

Comment:
I've never seen it recommended, but it is more explicit and transparent. I've started using ``list()``, ``dict()``, etc. rather than ``[]`` and ``{}`` in all of my code for that reason.